packages/core/src/components/Search/Search.tsx

```diff
@@ -19,6 +19,7 @@ const Search = forwardRef(
       clearIconName = CloseSmallIcon,
       clearIconLabel = "Clear",
       renderAction: RenderAction,
+      hideRenderActionOnInput = false,
```

Comment:
we're fine with it default to `undefined`

Reply:
updated, thanks @YossiSaadi